app/middleware/roleChecking.js

```diff
@@ -0,0 +1,35 @@
+const applicationRoles = require('../constants/application-roles-enum')
+
+const userNavigationOptions = {
+  dashboard: false,
+  claims: false,
+  audit: false,
+  download: false,
+  config: false
+}
+
+for (const role of Object.keys(applicationRoles)) {
```

Comment:
So having a bit more of a look I reckon you could do something like:

```
Object.keys(applicationRoles).forEach(role => {
...
})
```

more of a functional way of thinking.